### DISCUSSION TITLE
Adding C++ nodes , which can be of benefit to Ros 2 starters.

### DISCUSSION THREAD

**Discussion by Daviesss** [AhmettCelik1/Ros2BestPractice#7] [Ideas]
Hi , good repository you have got. I was wondering if i could send a pull request in adding some c++ node for Ros 2 beginners which can be of benefit. I will really appreciate your reply .

Regards.

**Comment by AhmettCelik1** [comment]
Hi, thank you for your compliment. Adding some nodes would be great. You
can send pull requests. Waiting for pull requests.

Davies iyanuoluwa Ogunsina ***@***.***>, 17 Haz 2023 Cmt,
01:39 tarihinde şunu yazdı:

**Reply by Daviesss** [reply]
Thanks for the response . Will send a request over the weekend.

Regards.
